var/spack/repos/builtin/packages/xios/package.py

```diff
@@ -12,10 +12,15 @@ class Xios(Package):
     """XML-IO-SERVER library for IO management of climate models."""
 
     homepage = "https://forge.ipsl.jussieu.fr/ioserver/wiki"
-
-    version("develop", svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/trunk")
     version(
-        "2.5", revision=1860, svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.5"
+        "2.6",
+        revision=2703,
+        svn="https://forge.ipsl.jussieu.fr/ioserver/svn/XIOS2/branches/xios-2.6",
+    )
+    version(
+        "2.5",
+        revision=1860,
+        svn="https://forge.ipsl.jussieu.fr/ioserver/svn/XIOS2/branches/xios-2.5",
     )
     version(
         "2.0", revision=1627, svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.0"
```

Comment:
Older versions can be marked as deprecated, and they can get removed after the next release. We typically avoid simply removing a version (even if you cannot verify that it compiles correctly since someone else on a different system may still be using the older version).

E.g.
```suggestion
    version(
        "2.5",
        revision=1860,
        svn="https://forge.ipsl.jussieu.fr/ioserver/svn/XIOS2/branches/xios-2.5",
        deprecated=True,
    )
    version(
        "2.0", revision=1627, svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.0"
```

Reply:
I'll deprecate the older versions this way. Thanks for the suggestion.

Reply:
Done